header.php

```diff
@@ -32,18 +32,16 @@
 			<?php if ( has_nav_menu( 'primary' ) ) { ?>
 				<nav id="site-navigation" class="primary-navigation" role="navigation" aria-label="<?php esc_attr_e( 'Primary', 'twentytwentyone' ); ?>">
 					<div class="menu-button-container">
-						<button id="primary-open-menu" class="button open" aria-haspopup="true" aria-controls="primary-navigation">
+						<button id="primary-mobile-menu" class="button" aria-haspopup="true" aria-controls="primary-navigation">
 							<span class="dropdown-icon open"><?php esc_html_e( 'Menu', 'twentytwentyone' ); ?>
 								<?php echo twenty_twenty_one_get_icon_svg( 'menu' ); // phpcs:ignore WordPress.Security.EscapeOutput ?>
 							</span>
-							<span class="hide-visually expanded-text"><?php esc_html_e( 'expanded', 'twentytwentyone' ); ?></span>
-						</button><!-- #primary-open-menu -->
-						<button id="primary-close-menu" class="button close" aria-haspopup="true" aria-controls="primary-navigation" aria-expanded="true">
 							<span class="dropdown-icon close"><?php esc_html_e( 'Close', 'twentytwentyone' ); ?>
 								<?php echo twenty_twenty_one_get_icon_svg( 'close' ); // phpcs:ignore WordPress.Security.EscapeOutput ?>
 							</span>
+							<span class="hide-visually expanded-text"><?php esc_html_e( 'expanded', 'twentytwentyone' ); ?></span>
```

Comment:
@aristath Should we not use aria-expanded instead of the expanded and collapsed text? But this can be a separate PR.

Reply:
Yes, using `aria-expanded` on the `button` element would be preferable... And if I'm not mistaken the "expanded" text won't be needed, as it is announced by screen-readers :+1: